common/addons/config-noise-function/src/main/java/com/dfsek/terra/addons/noise/samplers/noise/CellularSampler.java

```diff
@@ -351,6 +351,7 @@ public double getNoiseRaw(long sl, double x, double y) {
             case Distance2Mul -> distance1 * distance0 * 0.5 - 1;
             case Distance2Div -> distance0 / distance1 - 1;
             case NoiseLookup -> noiseLookup.noise(sl, centerX, centerY);
+            case OffsetNoiseLookup -> noiseLookup.noise(sl, centerX - x / frequency, centerY - y / frequency);
```

Comment:
The sampled positions (`coord / frequency`)  should be translated by the center# variables rather than the other way around, otherwise the coordinate space ends up being inverted (see value decreasing/getting darker towards the right rather than increasing):
![2023-09-21_10-18](https://github.com/PolyhedralDev/Terra/assets/73215501/7916ad4a-b7b0-4e5e-a3dc-46333d0c263b)

```suggestion
            case OffsetNoiseLookup -> noiseLookup.noise(sl, x / frequency - centerX, y / frequency - centerY);
```

Reply:
Good catch, updated it accordingly in the 2d and 3d variants